DigitalLibraryofTennessee/DLTN_XSLT
# Add missing roleTerms so that all name values are shared.

**GitHub Issue**: [248](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/248)

* Other Relevant Links - [mods_to_rdf 32](https://github.com/UTKcataloging/mods_to_rdf/pull/32)

## What does this Pull Request do?

This PR adds roleTerms that were missing from the current utkmodstomods transform. These were identified during analysis of UTK's use of the name top-level element for our MODS to RDF mapping.

## What's new?

The following roleTerms are new:

* Addressee
* Associated name
* Autographer
* Choreographer
* Costume designer
* Dedicatee
* Depicted
* Instrumentalist
* Music copyist
* Producer
* Signer
* Stage manager
* Surveyor
* Videographer

## How should this be tested?

Check for misspellings - no significant changes to the logic and functioning of the code were made. Double check that no terms are present as both Creator and Contributor.
